### DISCUSSION TITLE
Mod Multipliers for grow & deflate

### DISCUSSION THREAD

**Discussion by Nyanaro** [ppy/osu#25865] [Ideas]
The "Grow" and "Deflate" mods should be given mod multipliers when combined with the hidden mod, as they essentially add lesser approach circles for the hidden mod. I **don't** believe they should be disabled outright, as playing the game with one of the two as well as hidden is incredibly fun (and has become my preferred way of playing the game).

I suggest the mods to reduce the hidden multiplier from 1,06x to 1,03x when enabled alongside the hidden mod.

If the team prefers a static multiplier that applies to all mod combinations, I believe a 0,98x or 0,97x multiplier is reasonable. Kingling suggested 0,95x https://github.com/ppy/osu/discussions/25865#discussioncomment-7890527 which also makes sense. I would however not go lower than this under any circumstance, as Hidden+Grow/Deflate is still more difficult than nomod, so a multiplier lower than 0,95x would make picking grow+hd pointless from a leaderboard standpoint

**Comment by osuKingling** [comment]
IMO multipliers for the non-stable mod should be decided only once we have finalised multipliers for the stable mods as it's my understanding they're not locked in yet and would require us to change multipliers for all mods if they changed.

That aside, I think 0.95x would be best for these in their current state.

**Reply by Nyanaro** [reply]
That's a reasonable argument, I can stand behind that
